src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -133,7 +133,7 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
 
     public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         if (isRoeUpdateEnabled && overseasEntitySubmissionDto.isForUpdate()) {
-            // validatePartialUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+            validatePartialUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
This re-enables the validation taken out for uar-616 failures in reserve1/cidev.